pages/admin-panel.js

```diff
@@ -52,6 +52,9 @@ export const adminPanelQuery = gqlV2/* GraphQL */ `
           settings
         }
       }
+      ... on Collective {
+        isApproved
+      }
```

Comment:
Why not on `AccountWithHost`? You're missing FUNDs with this approach.

Reply:
Ah, thanks for pointing it out. I've corrected this. 👍🏽